kyma-project/busola
# Single frontend deployment

**Description**

Changes proposed in this pull request:

- Remove npx mode
- Use /backend path
- Remove `config.js` (the one that was used to get injected configmap)
- Serve all frontend MFs in single deployment, with common nginx config

Testing - deployment on a cluster:

- Deploy backend as usual (`kc apply -k .` in backend directory)
- Deploy `ingress--single.yaml` (remember to replace domain name), `deployment-single.yaml` and `service.yaml` from web directory)

Considerations:

- I've commented out adding licenses - as far as I could see, there were no licenses in previously built images.

TODOs:

- Add prow job for new image build
- After verifying how single deployment works on a prod, we can remove previous frontend deployments, Dockerfiles, nginx config etc (maybe cleanup Makefiles?)

**Related issue(s)**
